ratis-server/src/main/java/org/apache/ratis/server/impl/RaftServerImpl.java

```diff
@@ -1171,6 +1181,23 @@ public InstallSnapshotReplyProto installSnapshot(InstallSnapshotRequestProto req
     return reply;
   }
 
+  public boolean pause() throws IOException {
```

Comment:
pause() and resume() should be implemented together.  Otherwise, the feature is incomplete.

Reply:
Makes sense. Added resume() as well in this PR.